cmd/libs/go2idl/args/args.go

```diff
@@ -75,6 +79,16 @@ func (g *GeneratorArgs) LoadGoBoilerplate() ([]byte, error) {
 	return b, nil
 }
 
+// StableCommandLine returns the command that will invoke this generator; it
+// will be stable when the generator is run in verify mode. (For use if you
+// want to include the command line as instructions in the generated output
+// without making --verify-only fail.)
```

Comment:
I'm afraid I'm completely lost in this comment :( [to be honest I don't understand it]